seq2science/scripts/seurat/merge_seurat_objs.R

```diff
@@ -0,0 +1,67 @@
+suppressMessages({
+    library(Seurat)
+})
+
+# Snakemake variables
+seu_dir <- paste0(snakemake@config$result_dir,"/seurat/kallistobus")
+rds <- snakemake@output[[1]] 
+isvelo <- snakemake@params$isvelo
+log_file <- snakemake@log[[1]]
+seu_rdata <- snakemake@input$seu_objs   
+
+# log all console output
+log <- file(log_file, open="wt")
+sink(log)
+sink(log, type="message")
+
+# log all variables for debugging purposes
+cat('# variables used for this analysis:\n')
+cat('seu_dir      <- "', seu_dir,         '"\n', sep = "")
+cat('rds          <- "', rds,             '"\n', sep = "")
+cat('isvelo       <- "', isvelo,          '"\n', sep = "")
+
+cat('\n')
+
+# List RData files   
+print(seu_rdata)                             
```

Comment:
move to line 22 as 
```
cat('seu_rdata    <- c("', paste(seu_rdata, collapse='","'), '")\n', sep = "")
```

should look like this in the logs:
![image](https://user-images.githubusercontent.com/48289046/142609676-ef908d22-b07b-4699-b8bc-f2dc5a7102b1.png)

Reply:
Thanks for the tip 👍